eclipse-collections-api/src/main/java/org/eclipse/collections/api/collection/MutableCollection.java

```diff
@@ -138,7 +150,11 @@
      *
      * @see #removeAll(Collection)
      */
-    MutableCollection<T> withoutAll(Iterable<? extends T> elements);
+    default MutableCollection<T> withoutAll(Iterable<? extends T> elements)
+    {
+        this.removeAllIterable(elements);
+        return this;
+    }
```

Comment:
And these don't call `super`? I prefer these if I had to pick one, for consistency.